tests/config.spec.ts

```diff
@@ -19,7 +19,8 @@ import fs from 'node:fs';
 import { Config } from '../config.js';
 import { test, expect } from './fixtures.js';
 
-test('config user data dir', async ({ startClient, mcpBrowser }, testInfo) => {
+test('config user data dir', async ({ startClient, mcpMode }, testInfo) => {
```

Comment:
Introducing a `localOutputPath()` fixture that skips automatically will make all these tests nicer.